index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!-- the wraping container -->
+    <div class="parent-container">
+      <header class="header-section">
+        <!--under this are my logo and menu-->
+        <h1 class="logo"><a href="#">My Logo</a></h1>
+        <nav class="humberger-menu">
+          <a href="#"><i class="fa fa-bars fa-lg"></i></a>
+        </nav>
+      </header>
+      <!-- background image -->
```

Comment:
- [ OPTIONAL] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments.